Geometry/GlobalTrackingGeometryBuilder/python/globalTrackingGeometryESProducer_cfi.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+globalTrackingGeometryESProducer = cms.ESProducer( "GlobalTrackingGeometryESProducer" )
```

Comment:
This is not missing file - the bare producer is defined here:
http://cmslxr.fnal.gov/source/Geometry/CommonDetUnit/python/bareGlobalTrackingGeometry_cfi.py?v=CMSSW_7_4_X_2015-01-15-0200#0010